lib/routes/home.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:movie_db_app/widgets/error_message.dart';
+import '/routes/about_us.dart';
+import '/interfaces/i_repository.dart';
+import '/repositories/local/movie_repository_class.dart';
+import '/widgets/app_bar.dart';
+import '/widgets/display_movie_detail.dart';
+
+class Home extends StatefulWidget {
+  const Home({super.key});
+
+  static const double paddingInListView = 15;
+
+  @override
+  State<Home> createState() => _HomeState();
+}
+
+class _HomeState extends State<Home> {
+  final IMyRepository movieJsonManagement = MovieJsonManagement();
+  late Future<dynamic> moviesData;
+
+  final double dividerHeight = 50;
+  final double dividerIndent = 20;
+  final double endDividerIndent = 20;
+  final double dividerThickness = 5;
+  final String backgroundWhiteImagePath = "assets/images/white.png";
+
+  @override
+  void initState() {
+    moviesData = movieJsonManagement.getData();
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: MyAppBar(
+        appBar: AppBar(),
+      ),
+      body: FutureBuilder(
```

Comment:
Specify the type of data you are going to receive on the future builder

Reply:
solved